test/fixtures/linter/rules/NoGlobals/FormatterGlobalBindingString.js

```diff
@@ -12,7 +12,7 @@ sap.ui.define(
 		// Although the formatter property does not look like a global notation,
 		// it should still be detected if it is a string:
 		const input2 = new Input({
-			value: "{ path: 'invoice>Status', formatter: 'formatter.statusText' }"
+			value: "{ path: 'invoice>Status', formatter: formatter.statusText }"
```

Comment:
This causes a parsing error, which I think isn't the focus within this test fixture.